wrapper

```diff
@@ -188,19 +175,18 @@ if __name__ == "__main__":
     job_failed = False
 
     # Run the job.
-    output_files = []
     try:
         updater.running("processing the input files")
-        output_files = run_job(cfg.arguments, cfg.stdout, cfg.stderr)
+        run_job(cfg.arguments, cfg.stdout, cfg.stderr)
     except Exception as e:
         updater.running("job encountered an error: {0}".format(e))
         job_failed = True
 
     # Upload the output files.
     try:
         updater.running("uploading the output files")
-        files_to_upload = [cfg.stdout, cfg.stderr] + output_files
-        upload_files(cfg.output_ticket_list, cfg.irods_user, cfg.irods_job_user, files_to_upload, updater)
+        paths = get_paths_to_upload(get_excluded_files(cfg))
```

Comment:
I just realized that I forgot to include the input files in the list of files to exclude. I'll push a change to fix that momentarily.